packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -305,123 +387,173 @@ describe('meeting', () => {
   });
 
   describe('getLiveStreamState', () => {
+    const allowedContexts = [FrameContexts.sidePanel];
+    it('should fail when called without a callback', () => {
```

Comment:
nit: can you change this test case to say 'should fail when called  with a null callback'.

It's harder to distinguish between "without a callback" and "empty callback", IMO.